recipes/cppcommon/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
```

CXX 17 was added in 3.8
```suggestion
cmake_minimum_required(VERSION 3.8)
```